openweathermap-ios/openweathermap-ios/MapViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  MapViewController.swift
+//  openweathermap-ios
+//
+
+
+import UIKit
+import MapKit
+
+class MapViewController: UIViewController, CLLocationManagerDelegate {
+
+    @IBOutlet weak var mapView: MKMapView!
+    let locationManager = CLLocationManager()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        mapView.delegate = self
+        locationManager.delegate = self
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        locationManager.requestWhenInUseAuthorization()
+        locationManager.startUpdatingLocation()
+        
+    }
+    
+    
+    func addAnnotation(location: CLLocationCoordinate2D){
+        print("Coordenate center: \(mapView.centerCoordinate)")
+        let circleCenter = MKCircle(center: location, radius: 50)
+        let circleEdge = MKCircle(center: location, radius: 5)
+        
+        mapView.overlays.forEach { (overlay) in
+            if (overlay is MKCircle){
```

Comment:
```suggestion
            if overlay is MKCircle {
```

Don't need parentheses on if statements.